src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $color-gray;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0 auto 4px;
+    color: $color-dark;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    color: $color-lighter;
+    margin: 0 0 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__review-number {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-dark;
+    margin-top: 9px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 16px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $color-lighter;
+    margin: 0;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    color: $color-dark;
+    font-size: 16px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__buy-button {
+    width: 100%;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 38px;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    text-transform: uppercase;
+    background: $color-background;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      background-color: #fff;
```

Comment:
Repeatable code move to variables
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/85a99418-becf-402b-bf51-0ebc3c3ffefa)